spec/spec_helper.rb

```diff
@@ -1,19 +1,9 @@
 require 'simplecov'
 require 'simplecov-lcov'
 
+core_root = './lib/chess/core/'
 require 'byebug'
-require './lib/chess/chess'
-require './lib/chess/pieces/piece'
-require './lib/chess/pieces/king'
-require './lib/chess/pieces/queen'
-require './lib/chess/pieces/bishop'
-require './lib/chess/pieces/knight'
-require './lib/chess/pieces/rook'
-require './lib/chess/pieces/pawn'
-require './lib/chess/position'
-require './lib/chess/board/board'
-require './lib/chess/book'
-require './lib/chess/prompt'
+require "#{core_root}chess"
 
```

Comment:
still seeing here this `require`

Reply:
but, deleted no?

Reply:
<img width="726" alt="image" src="https://user-images.githubusercontent.com/26731448/204640736-392da53f-e653-4b4d-8687-1506519af6df.png">